brainfuck/interpreter.go

```diff
@@ -0,0 +1,85 @@
+package brainfuck
+
+/*
+	Interpret(code string) interprets brainfuck code string and prints
```

Comment:
```suggestion
	Interpret interprets brainfuck code string and prints
```
Don't need to repeat the signature, just the name.

Reply:
fixed